dbt_style_guide.md

```diff
@@ -396,6 +396,15 @@ For more information about why we use so many CTEs, check out [this glossary ent
           └── stg_jaffle_shop__payments.sql
   ```
 
+- `dbt_project.yml` configurtions should be prefixed with `+` to avoid namespace collision with directories.

```

Comment:
Just a tiny typo

```suggestion
- `dbt_project.yml` configurations should be prefixed with `+` to avoid namespace collision with directories.

```

Reply:
Thank you for catching @b-per